packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -48,6 +48,18 @@ export class HbarLimitService implements IHbarLimitService {
    */
   private readonly hbarLimitRemainingGauge: Gauge;
 
+  /**
+   * Tracks the number of unique spending plans used daily.
```

Comment:
Well, if a user that is linked to the plan makes any request in that day, the plan is considered to be "used" for that day.

There might be a better word for it in English instead of "use" but I'm not sure what it might be 😄.

Reply:
Changed the comment to:
```typescript
  /**
   * Tracks the number of unique spending plans that have been utilized on a daily basis
   * (i.e., plans that had expenses added to them).
   *
   * For basic spending plans, this equates to the number of unique users who have made requests on that day,
   * since each user has their own individual spending plan.
   *
   * @private
   */
  private readonly dailyUniqueSpendingPlansCounter: Record<SubscriptionType, Counter>;
```

I hope that is clearer and makes more sense!

Reply:
yep the `For basic spending plans, this equates to the number of unique users who have made requests on that day,  since each user has their own individual spending plan.` explains it well! Thanks much for updating it!